afs/afs-ext-base/src/main/java/com/powsybl/afs/ext/base/ReadOnlyScript.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2017, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.afs.ext.base;
+
+/**
+ *
+ * @author Nicolas Lhuillier <nicolas.lhuillier at rte-france.com>
+ */
+public interface ReadOnlyScript {
+
+    default String getScriptLabel() {
+        return null;
+    }
+
+    ScriptType getScriptType();
+
+    String readScript();
```

Comment:
It's weird to be able to read a script from the storage without be able to store the script... How the script could exist in the storage without a writeScript feature?